js/src/redux/reducers/openSidebarSections.js

```diff
@@ -0,0 +1,49 @@
+import { CLOSE_ALL_SIDEBAR_SECTIONS, OPEN_SIDEBAR_SECTION, CLOSE_SIDEBAR_SECTION } from "../actions/openSidebarSections";
+
+const INITIAL_STATE = [];
+
+/**
+ * Helper function: returns an array with the requested item removed.
+ *
+ * @param   {Array}  openSections The array from which the item should be removed.
+ * @param   {string} removeItem   The item to be removed.
+ *
+ * @returns {Array}               The array without the item that should be removed.
+ */
+let sidebarSectionCloser = ( openSidebarSections, removeSection ) => {
+	return openSidebarSections.filter( item => item !== removeSection );
+};
+
+let sidebarSectionOpener = ( openSidebarSections, addSection ) => {
+	if ( typeof addSection !== "string" || openSidebarSections.includes( addSection ) ) {
+		return openSidebarSections;
+	}
+
+	return [
+		...openSidebarSections,
+		addSection,
+	];
+};
+
+/**
```

Comment:
Don't make it more elaborate than it needs to be ;)
```
/**
 * A reducer for adding and removing section ids.
 *
 * @param {string} state The current state of the object.
 * @param {Object} action The current action received.
 *
 * @returns {string} The state.
 */
```

Reply:
While I agree I went a little overboard here, what I was going for was documenting which part of the state this reducer actually manages. In this file you cannot see this anywhere, you'd have to visit the rootReducer in edit.js. To have everything together, it could be worthwhile to write down the name of the state that is managed per reducer.